homebrew/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+NVS_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+
+# Set environment variables
+export HOME=~
+export NVS_HOME=~/.nvs
+
+# Link necessary files
+ln -fs $NVS_DIR/nvs.sh $NVS_HOME/nvs.sh
+ln -fs $NVS_DIR/lib $NVS_HOME/lib
+ln -fs $NVS_DIR/deps $NVS_HOME/deps
+ln -fs $NVS_DIR/package.json $NVS_HOME/package.json
+ln -fs $NVS_DIR/defaults.json $NVS_HOME/defaults.json
+
+# Run install script
+\. $NVS_DIR/nvs.sh install
```

Comment:
Homebrew downloads the repo files to somewhere under `/opt/homebrew`, which I set as the variable `NVS_DIR`. Linking the needed files to `NVS_HOME`, which I defaulted to `$HOME/.nvs`, will essentially make it as if we cloned the repo in `NVS_HOME`. This way, the auto-generated export lines will work as intended. If we don't do the linking, we might need to make some changes to the product code so that export lines taking `NVS_ROOT` into consideration will be used. 

I think since the environment variables are set correctly, there's no difference in using `$NVS_DIR/nvs.sh` or `$NVS_HOME/nvs.sh` as one of them is a soft link of the other. 

Coming to think of it, making some changes to the product code so that the export lines take account of `NVS_ROOT` might be the cleanest option. We might also allow for skipping downloading the bootstrap Node.js in `nvs.sh install` and postponing it to when the user first run the `nvs` command. This should allow the user to choose their preferred `NVS_HOME` and not have to `mv ~/.nvs /somewhere/else` if they want a different location for `NVS_HOME`.

Reply:
I'll take a look at the code tonight. Let's not check in this PR before I attempt at the product changes to make installation clean through Homebrew.

Reply:
OK. I made some changes so that the shell scripts generated respect the case where NVS_HOME is not the install directory. The script will use the directory where `nvs.sh` is located if it detects that $NVS_HOME is not where `nvs.sh` is. Let me know if you have further suggestions.